<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogOptions</class>
 <widget class="QDialog" name="DialogOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Book</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Title</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditTitle"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Export</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QFormLayout" name="formLayout_2">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Device</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="comboBoxDevice">
          <item>
           <property name="text">
            <string>Kindle 1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Kindle 2/3/Touch</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Kindle 4 &amp; 5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Kindle DX/DXG</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Kindle Paperwhite 1 &amp; 2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Kindle Paperwhite 3/Voyage/Oasis</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Kobo Mini/Touch</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Kobo Glo</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Kobo Glo HD</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Kobo Aura</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Kobo Aura HD</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Kobo Aura H2O</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelFormat">
          <property name="text">
           <string>Format</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="comboBoxFormat">
          <item>
           <property name="text">
            <string>Images &amp; CBZ &amp; PDF</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Images only</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>PDF only</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>CBZ only</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="checkboxOverwrite">
        <property name="text">
         <string>Overwrite existing files</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkboxOrient">
        <property name="text">
         <string>Orient images to match aspect ratio</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkboxQuantize">
        <property name="text">
         <string>Dither images to match device palette</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkboxFrame">
        <property name="text">
         <string>Draw frame around images</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkboxSplit">
        <property name="text">
         <string>Split images into two pages (right, left)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkboxSplitInverse">
        <property name="text">
         <string>Split images into two pages (left, right)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkboxAutoCrop">
        <property name="text">
         <string>Auto crop image (remove white around the image)</string>
        </property>
       </widget>
      </item>
      <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Size</string>
          </property>
         </widget>
        </item>
      <item>
       <widget class="QRadioButton" name="checkboxResize">
        <property name="text">
         <string>Resize images to center on screen</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="checkboxStretch">
        <property name="text">
         <string>Stretch images to fill the screen</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogOptions</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogOptions</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
